### DISCUSSION TITLE
Remove pager from front page

### DISCUSSION THREAD

**Discussion by mcmcclur** [observablehq/framework#1647] [Q&A] [answered]
I'd like to remove the paging feature from just the front page of my Framework project. I might think that adding

    pager: false

within the `---` delimited YAML frontmatter in `src/index.md` file would accomplish that but it doesn't seem to. I can hide it with CSS but wondered if there was a more direct way to accomplish this?

**Comment by mbostock** [comment, marked as answer]
It’s possible to do this with the **pager** field and the [**pages** config option](https://observablehq.com/framework/config#pages). The **pager** field declares the page’s page group, and all pages within the same page group are linked together via the footer. The default page group is `main`, and the home page `/index` is always in the `main` page group. Therefore, if you declare a different page group for all your other pages, say `alt`, you’ll effectively disable the pager for the home page since the `main` page group will only have a single page in it. That might look like this in your `observablehq.config.js`:

```js
export default {
  pages: [
    {name: "Page one", pager: "alt", path: "/one"},
    {name: "Page two", pager: "alt", path: "/two"},
    {name: "Page three", pager: "alt", path: "/three"}
  ]
};
```

It’s a bit awkward to repeat the `pager` for every page… but it works. 😅 

If you don’t mind that the first page (above `/one`) will have a previous link back to the main page, another option is to hide the pager on the main page. You can do that with CSS:

```html
<style type="text/css">

#observablehq-footer nav {
  display: none;
}

</style>
```

There was some previous discussion of this in #201, but we could open up a new issue for some front matter control over the page.

**Reply by mcmcclur** [reply]
Thanks Mike - that actually works great for my use case! I've got a main page, an about page, and then a number of samples. This makes it easy to link all the samples together and leave those other two pages out of it. Perfect!
